Library/Homebrew/tap.rb

```diff
@@ -112,6 +112,16 @@ def to_hash
     }
   end
 
+  def load_tap_formula_renames
```

Comment:
Might have sense, thanks.
I used the same structure as `tap_migrations.rb` has. Btw, I know about that warnings and thought it doesn't really matter.